documentation/polymer-templates/tutorial-template-basic.asciidoc

```diff
@@ -1,24 +1,56 @@
 ---
-title: Creating a Simple Component Using the PolymerTemplate API
+title: Creating a Simple Component Using the Template API
 order: 2
 layout: page
 ---
 
 ifdef::env-github[:outfilesuffix: .asciidoc]
 
-= Creating a Simple Component Using the PolymerTemplate API
+= Creating a Simple Component Using the Template API
 
-In this section, we demonstrate how to create a simple component using only the `PolymerTemplate` API.
+In this section, we demonstrate how to create a simple component using only the Template API.
 
-Our example:
+It's recommended to use LitElement template language for your components, but PolymerTemplate is also supported.
 
-* Creates a simple view that allows the user to input their name and click a button for a response.
+Our example creates a simple view that allows the user to enter their name and click a button.
 
-* Uses standard Polymer 3.0 features. See https://www.polymer-project.org/3.0/docs/about_30[Polymer Library 3.0] to learn more.
+[NOTE]
+This documentation covers LitElement, Polymer 3 and npm in Vaadin version 14+ only. Earlier Vaadin versions support Polymer 2, HTML imports, and the Bower package manager. You can still use these components in Vaadin 14, if you work in the special *compatibility mode*. See the https://vaadin.com/docs/v13/flow/polymer-templates/tutorial-template-basic.html[Vaadin 13 documentation] for how to build client-side components with Polymer 2 and HTML imports.
 
+== Creating the Template File on the Client Side
 
+=== LitElement
 
-== Creating the Template File on the Client Side
+The first step is to create the LitElement JavaScript template file on the client side in `frontend/src/hello-world.js`. This file contains the view structure.
+
+*Example*: Creating the `hello-world.js` JavaScript LitElement template file.
+
+[source,js]
+----
+import {LitElement, html} from 'lit-element';
+import '@polymer/paper-input/paper-input.js';
+
+class HelloWorld extends LitElement {
+
+    render() {
+        return html`
+            <div>
+                <paper-input id="inputId"></paper-input>
```

Comment:
Should use `vaadin-text-field` and `vaadin-button` instead?

Reply:
Done.